43-people-per-household-2014-2015.csv

```diff
@@ -0,0 +1,53 @@
+id,stateAbbr,stateName,2013,2014
```

Comment:
`2013` and `2014` would be better named `peoplePerHousehold2013` and `peoplePerHousehold2014`. It makes it clearer what this dataset is in case someone ends up using it without the `README.md` or the filename is changed. Also, integers as data labels could cause issues when the data is used programmatically.